### PR TITLE
RANCHER-1336: Add Keycloak Service Deployment

### PR DESCRIPTION
**Pull request by yaroslavishchenko**
Related Jira Ticket:
[RANCHER-1336: Add keycloak deployment to Rancher project provisioning](https://folio-org.atlassian.net/browse/RANCHER-1336)

### CODE REVIEW

File: terraform/rancher/project/keycloak.tf [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  version    = "21.0.4"
+  repository = "https://repository.folio.org/repository/helm-bitnami-proxy"
+  values = [
+    <<-EOF
+image:
+  registry: 732722833398.dkr.ecr.us-west-2.amazonaws.com
+  repository: folio-keycloak
+  tag: latest
+  pullPolicy: Always
+  debug: true
```
Comment (OHaimanov) [inline terraform/rancher/project/keycloak.tf:21]: Do we need debug option for permanent solution?

File: terraform/rancher/project/keycloak.tf [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+ingress:
+  enabled: true
+  hostname: ${local.keycloak_url}
+  ingressClassName: ""
+  pathType: ImplementationSpecific
+  path: /*
+  annotations:
+    kubernetes.io/ingress.class: alb
+    alb.ingress.kubernetes.io/scheme: internet-facing
+    alb.ingress.kubernetes.io/group.name: folio-tmp.test-2
```
Comment (OHaimanov) [inline terraform/rancher/project/keycloak.tf:170]: I think that this part should be renderd like in minio to point to correct group cluster/ns

File: terraform/rancher/project/keycloak.tf [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+    alb.ingress.kubernetes.io/listen-ports: '[{"HTTP":80}]'
+    alb.ingress.kubernetes.io/success-codes: 200-399
+    alb.ingress.kubernetes.io/healthcheck-path: /realms/master
```
Comment (OHaimanov) [inline terraform/rancher/project/keycloak.tf:173]: maybe here should be /health/ready or live?

**Review by OHaimanov — CHANGES_REQUESTED** [review]

File: terraform/rancher/project/keycloak.tf (hunk 1 above)
Comment (yaroslavishchenko) [reply terraform/rancher/project/keycloak.tf:21]: This "Debug" option is used for BITNAMI docker image and init code inside.
Here we can disable it.

File: terraform/rancher/project/keycloak.tf (hunk 2 above)
Comment (yaroslavishchenko) [reply terraform/rancher/project/keycloak.tf]: Thank You.
Forgot to remove debug value for toubleshooting.

**Review by OHaimanov — APPROVED** [review]

**Review by aatoyan — APPROVED** [review]

**Pull request merged**